charts/kong/ci/custom-labels.yaml

```
@@ -0,0 +1,6 @@
+
+# install chart with custon labels
+
+labels: {
+  acme.com/environment: development
+}
```

```suggestion
labels:
  acme.com/environment: development
```

We don't use curly braces for dicts, let's remove them for style consistency.

---

Ok